dev-php/jsonlint/jsonlint-1.4.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+DESCRIPTION="JSON Lint for PHP"
+HOMEPAGE="https://github.com/Seldaek/jsonlint"
+SRC_URI="https://github.com/Seldaek/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="dev-lang/php:*
+	=dev-php/fedora-autoloader-0.2.1"
+DEPEND="test? ( ${RDEPEND}
+				dev-php/phpunit )"
```

Comment:
Ok I understand, I'll close the brace on the same line as last text, to remain consistent and short, and close the brace indented with the opening on a new line for the unusual/complex cases.